js/alumni.js

```diff
@@ -143,6 +143,14 @@ var colleges = [
         },
         image: "images/collegeLogos/University_of_Padua.png"
     },
+    {
+        name: 'Vanderbilt University',
+        position: {
+            lat: 36.1446,  
+            lng: -86.8026,
+        },
+        image: "images/collegeLogos/Vanderbilt_University.png"
```

Comment:
Use consistent quotes

Reply:
what do you mean? as far as I can tell all the images use double quotes

Reply:
Ah you're right—but we should standardize on one for the entire document. Not necessary to do in this PR.